src/modules/hideSurveyPopup.js

```diff
@@ -0,0 +1,14 @@
+const $ = window.jQuery;
+const utils = require("../utils");
+
+/* Hide popup notifying the user about unfilled surveys on every login */
```

Comment:
Minor: with the 2.0 refactor I've changed the comment style to `//` everywhere else in the code base. Could you also change this one?